src/dataset.js

```diff
@@ -202,6 +203,30 @@ export class Dataset {
         return datasets.getItems(params);
     }
 
+    /**
+     * Returns a dataset object containing general information about the dataset.
```

Comment:
"Returns a dataset object" ?

Reply:
I got my inspiration here: https://www.apify.com/docs/sdk/apify-client-js/latest#ApifyClient-datasets-getDataset

Perhaps just "an object" ?

Reply:
Yeah, "an object" will do, we need to improve our naming.